presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -803,6 +805,22 @@ protected Type visitFunctionCall(FunctionCall node, StackableAstVisitorContext<C
                 throw e;
             }
 
+            if (node.getSortItems().isPresent()) {
```

Comment:
I have run th query and it seems that for query:
```
explain select 1 as c, nationkey from customer group by nationkey order by sum(custkey order by c);
```
`ORDER BY c` is left untranslated in `com.facebook.presto.sql.planner.QueryPlanner#aggregate`.
This is rather a but that analysis passed on such query and planner tries to plan it. Definitely you should explicitly fail in `AggregationAnalyzer` with a proper error message